File: dotnet/Potter.Tests/PotterTests.cs
using System.Collections.Generic;
using Xunit;

namespace Potter.Tests;

public class PotterTests
{
    [Fact]
    public void Should_Work()
    {
        Assert.True(true);
    }

    [Fact]
    public void Should_Not_Work()
    {
        var potter = new Potter();
        Assert.Equal(0, potter.GetPrice(new List<int>()));
    }
}

File: java/src/test/java/PotterTests.java
import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.api.Test;

public class PotterTests {

    @Test
    public void Should_Work() {
        Assertions.assertTrue(true);
    }

    @Test
    public void Should_Not_Work() {
        Potter potter = new Potter();
        Assertions.assertEquals(0, potter.getPrice(new int[]{0}));
    }
}
